src/OXRS_Rack32.cpp

```diff
@@ -109,16 +109,45 @@ void _getConfigSchemaJson(JsonVariant json)
     _mergeJson(properties, _fwConfigSchema.as<JsonVariant>());
   }
 
-  // Rack32 config
+  // MCP9808 temp sensor config
   if (_tempSensorFound)
   {
     JsonObject temperatureUpdateSeconds = properties.createNestedObject("temperatureUpdateSeconds");
     temperatureUpdateSeconds["title"] = "Temperature Update Interval (seconds)";
-    temperatureUpdateSeconds["description"] = "How often to read and report the value from the onboard MCP9808 temperature sensor, in seconds (defaults to 60 seconds, 0 will disable temperature reports). Must be a number between 0 and 86400 (i.e. 1 day).";
+    temperatureUpdateSeconds["description"] = "How often to read and report the value from the onboard MCP9808 temperature sensor (defaults to 60 seconds, setting to 0 disables temperature reports). Must be a number between 0 and 86400 (i.e. 1 day).";
     temperatureUpdateSeconds["type"] = "integer";
     temperatureUpdateSeconds["minimum"] = 0;
     temperatureUpdateSeconds["maximum"] = 86400;
   }
+
+  // LCD config
+  JsonObject activeBrightnessPercent = properties.createNestedObject("activeBrightnessPercent");
+  activeBrightnessPercent["title"] = "LCD Active Brightness (%)";
+  activeBrightnessPercent["description"] = "Brightness of the LCD when active (defaults to 100%). Must be a number between 0 and 100.";
+  activeBrightnessPercent["type"] = "integer";
+  activeBrightnessPercent["minimum"] = 0;
+  activeBrightnessPercent["maximum"] = 100;
+ 
+  JsonObject inactiveBrightnessPercent = properties.createNestedObject("inactiveBrightnessPercent");
+  inactiveBrightnessPercent["title"] = "LCD Inactive Brightness (%)";
+  inactiveBrightnessPercent["description"] = "Brightness of the LCD when in-active (defaults to 10%). Must be a number between 0 and 100.";
+  inactiveBrightnessPercent["type"] = "integer";
+  inactiveBrightnessPercent["minimum"] = 0;
+  inactiveBrightnessPercent["maximum"] = 100;
+
+  JsonObject activeDisplaySeconds = properties.createNestedObject("activeDisplaySeconds");
+  activeDisplaySeconds["title"] = "LCD Active Display (seconds)";
+  activeDisplaySeconds["description"] = "How long the LCD remains 'active' after an event is detected (defaults to 10 seconds). Must be a number between 0 and 600 (i.e. 10 minutes).";
```

Comment:
Should be mentioned that 0 turns on forever (disables timer)

Reply:
Ok - will add that

Reply:
@moinmoin-sh - pushed an update - changed the title and description - please review.